src/HttpClientFactory/Http/src/DependencyInjection/HttpClientFactoryServiceCollectionExtensions.cs

```diff
@@ -52,6 +52,12 @@ public static IServiceCollection AddHttpClient(this IServiceCollection services)
             // because we access it by reaching into the service collection.
             services.TryAddSingleton(new HttpClientMappingRegistry());
 
+            //Register default client as HttpClient
+            services.AddTransient(s =>
```

Comment:
Just spotted this PR and I wonder, should it use `TryAddTransient()` instead?

Existing apps might already manually configure this in pre-5.0.0 (I've definitely done it in places), and then when upgraded to 5.0.0 forget to remove it and then have two `HttpClient` dependencies registered instead of one?

Reply:
I'd be OK with that change, it's definitely an improvment. Want to send a PR?

Reply:
Sure, I’ll fire one up in a bit.

Reply:
#2131